packages/xdl/src/detach/IosNSBundle.js

```diff
@@ -366,6 +366,13 @@ async function _configureInfoPlistAsync(context: StandaloneContext) {
       infoPlist.UIDeviceFamily = [2];
     }
 
+    // Whether requires full screen on iPad
+    infoPlist.UIRequiresFullScreen = config.ios && config.ios.requireFullScreen;
+    if (infoPlist.UIRequiresFullScreen === null || infoPlist.UIRequiresFullScreen === undefined) {
```

Comment:
We tend to use `infoPlist.UIRequiresFullScreen == null`, which does the same check and handles undefined.